docs/fundamentals/code-analysis/quality-rules/ca1505.md

```diff
@@ -72,6 +72,28 @@ dotnet_analyzer_diagnostic.category-Maintainability.severity = none
 
 For more information, see [How to suppress code analysis warnings](../suppress-warnings.md).
 
+## Configure threshold
+
+You can configure the threshold at which this rule fires.
+
+1. Create a text file. As an example, you can name it *CodeMetricsConfig.txt*.
+
+2. Add the desired threshold to the text file in the following format:
+
+   ```txt
+   CA1505: 10
+   ```
+
+   In this example, the rule is configured to fire when a method's maintainability index is less than 10.
```

Comment:
What is the default threshold for CA1505?

Reply:
Default thresholds are defined here: https://github.com/dotnet/roslyn-analyzers/blob/63367165996d9dedbecf6677e3315a470a2f557c/src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/CodeMetricsAnalyzer.cs#L259-L280